paperqa/clients/client_models.py

```diff
@@ -115,7 +116,7 @@ async def query(self, query: dict) -> DocDetails | None:
                 f"{client_query.doi if isinstance(client_query, DOIQuery) else client_query.title}"
                 f" in {self.__class__.__name__}."
             )
-        except TimeoutError:
+        except (TimeoutError, asyncio.exceptions.TimeoutError):
```

Comment:
Can you add a comment here saying after Python>=3.11 `TimeoutError` replaces `asyncio.exceptions.TimeoutError`